pkg/virt-launcher/virtwrap/converter/converter.go

```diff
@@ -1698,33 +1682,59 @@ func Convert_v1_VirtualMachineInstance_To_api_Domain(vmi *v1.VirtualMachineInsta
 	return nil
 }
 
+func detectEFICodeRom(c *ConverterContext, secureBoot bool) (string, error) {
```

Comment:
It is best to test these functions independent with their own tests and using the converter context. This way the converter can  be fully focused on the transformation and these functions can fully focus on e.g. mocking the filesystem to gather the information.

Reply:
Finally this info is static. Consider gathering it once during the manager startup.